chapters/05-funcoes-de-ordem-superior.md

```diff
@@ -30,13 +30,13 @@ Qual é mais propenso a erros?
 
 Se juntarmos as definições de `sum` e `range` o tamanho do segundo código será bem maior do que o primeiro, mas eu diria que ele é o mais correto.
 
-A razão dele ser o mais correto é que primeiramente ele constrói uma linguagem que expressa o problema e resolve em seu próprio domínio com uma solução clara. Somar uma sequência de números não se trata de um laço com contadores mas sim de somas e sequências de números.
+A razão dele ser o mais correto é que primeiramente ele constrói uma linguagem que expressa o problema e resolve em seu próprio domínio com uma solução clara. Somar uma sequência de números não se trata de um laço com contadores. Trata-se de intervalos e somas.
```

Comment:
**Original:**
*It is more likely to be correct because the solution is expressed in a vocabulary that corresponds to the problem being solved. Summing a range of numbers isn’t about loops and counters. It is about ranges and sums.*

**Tradução:**
A razão dele possivelmente ser o mais correto, é que a solução é expressa em um vocabulário que corresponde ao problema que está sendo resolvido. Somar um intervalo de números não se trata de laços de repetição e contadores. Trata-se de intervalos e somas.